### DISCUSSION TITLE
Transparent track - no map

### DISCUSSION THREAD

**Discussion by NicolayFlaaten** [gpx-animator/gpx-animator#524] [General]
Is it possible to have a transparent background so I can position the track above a picture of another map or as a part of a video in my video editor?

Thank you for a great tool!

**Comment by McPringle** [comment]
No, that is not possible. Videos (h.264, h.265, etc) do not support transparency. You can either select your image as a background or you can specify a solid colour for the background and create a green-screen effect with your video editor.

**Comment by KyleGW** [comment]
You can do this with https://github.com/time4tea/gopro-dashboard-overlay and the Circuit map widget, generating a overlay only. 

https://github.com/time4tea/gopro-dashboard-overlay/tree/main/docs/xml/examples/06-circuit-map
